Frontend/ui-customer/src/Screens/CustomerUtils/AddNewAddressScreen/Controller.purs

```diff
@@ -360,6 +375,8 @@ getSavedLocations savedLocation =  (map (\ (SavedReqLocationAPIEntity item) ->
 , alpha : 1.0
 , fullAddress : getAddressFromSaved (SavedReqLocationAPIEntity item)
 , locationItemType : Just SAVED_LOCATION
+, distance : ""
```

Comment:
@prashant601 make distance as Maybe String, so that instead Nothing can be used instead of ""